tests/dummy/app/routes/home.js

```diff
@@ -1,8 +1,11 @@
 import Route from '@ember/routing/route';
-import InfinityRoute from 'ember-infinity/mixins/route';
+import { get } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default Route.extend({
+  infinity: service(),
 
-export default Route.extend(InfinityRoute, {
   model() {
-    return this.infinityModel('post');
+    return get(this, 'infinity').model('post');
```

Comment:
move all dummy routes to use service instead of route mixin.  Still have route mixin unit tests though.